syncpack.config.cjs

```diff
@@ -0,0 +1,195 @@
+/*!
```

Comment:
We should probably document (somewhere) what the process is / what the expectations are for developers to make changes here. What should a dev do if they run an error due to this tooling?

Reply:
Agreed, but since it's not enabled -- you have to manually run it -- this PR won't affect anyone. Once we're sure this works well for us, then we can follow up to operationalize it as part of CI.